app/models/app_config.rb

```diff
@@ -12,6 +12,9 @@ class AppConfig < ApplicationRecord
   USE_AUTH0_FOR_NEW_USERS = 'use_auth0_for_new_users'.freeze
   # The maximum number of samples that can be part of one bulk download.
   MAX_SAMPLES_BULK_DOWNLOAD = 'max_samples_bulk_download'.freeze
+  # The maximum number of samples that can be part of an original input files bulk download.
+  # Original input file downloads are significantly bigger and slower than other downloads, so a separate limit is needed.
+  MAX_SAMPLES_BULK_DOWNLOAD_ORIGINAL_FILES = 'max_samples_bulk_download_original_files'.freeze
```

Comment:
I'd love to see here what the initial values are and how they were chosen, for both configs.

Reply:
Max samples will be 1000 and max samples original files will be 100. This was based on stress tests which you can find here: https://czi.quip.com/gYUwA4tL0aIr/Bulk-Download-Scalability#faOACABnDFb

I considered how many samples the projects in IDseq contained. I wanted users to be able to bulk download data for all samples in their project for the vast majority of projects, instead of having to split their downloads up. The stress tests verified that the size and download time for the files was roughly reasonable.

Original files is much smaller because based on testing, it tends to be much bigger and slower than other downloads. Also, original files seems to be a bit of a special case. (it's uploader-only, etc.)